ludwig/schema/model_config.py

```diff
@@ -511,7 +511,7 @@ def _set_hyperopt_defaults(self):
         if not self.hyperopt:
             return
 
-        scheduler = self.hyperopt.get("executor", {}).get("scheduler")
+        scheduler = self.hyperopt.get("executor", {}).get("scheduler", {})
```

Comment:
I did this initially because I found that it was possible for the `scheduler` key to not be inside the `executor` dict, but that's no longer the case once we populate `self.hyperopt` from the `HyperoptConfig` so I will remove it

Reply:
Even if `scheduler` is not in the dict, it will just return `None`.